x/bank/keeper/keeper.go

```diff
@@ -21,6 +21,7 @@ var _ Keeper = (*BaseKeeper)(nil)
 // between accounts.
 type Keeper interface {
 	SendKeeper
+	WithMintCoinsRestriction(MintingRestrictionFn) BaseKeeper
```

Comment:
We're changing a public API so this is API changing. When resolving conflicts in Changelog, could you move the `Add `bank.BaseKeeper.WithMintCoinsRestriction` function to restrict use of bank `MintCoins` usage.` part into the API-breaking section?

Reply:
Gotcha!